src/App.tsx

```diff
@@ -15,58 +15,74 @@ export const goods = [
   'Garlic',
 ];
 
-export const App: React.FC = () => (
-  <main className="section container">
-    <h1 className="title">No goods selected</h1>
+export const App: React.FC = () => {
+  const [choosenProduct, setChoosenProduct] = useState<string>(`Jam`);
```

Comment:
If you add default value then typescript will infer the type of state without problems.